src/services/AccessTokenRequestProcessor.ts

```diff
@@ -52,7 +52,7 @@ export class AccessTokenRequestProcessor {
 			let session: ISessionItem | undefined;
 			try {
 				session = await this.f2fService.getSessionByAuthorizationCode(requestPayload.code);
-				this.logger.info({ message: "Found Session: " + JSON.stringify(session) });
+				this.logger.info({ message: "Found Session for AccessTokenRequestProcessor" });
```

Comment:
Do we want to move this log down a couple of lines below the `if (!session) {` otherwise we are potentially logging a success message when we actually have no session?

Could we also please add a log to the `if (!session) {` to let us know that the session wasn't found?